apps/dashboard/src/contexts/configured-chains.tsx

```diff
@@ -1,11 +1,13 @@
-import { type Chain, defaultChains } from "@thirdweb-dev/chains";
+// to remove @jonas (2/2)
+import { defaultChains } from "@thirdweb-dev/chains";
```

Comment:
where is this used? part of the configured chains?

Reply:
This one is being used in a Context (ChainsProvider) which in turn is being used in bunch of other places. My goal will be to change the code so that it works with type ChainMetadata (v5) instead of Chain (v4)

Reply:
okay, you can *try* as part of this simply removing the `defaultChains` - we'll have to test how that behaves / if it works. The concept of "defaultChains" is completely outdated regardless so I feel like it should work simply not pre-filling that state with it. (can instead pass an empty array to the state if types complain)

Reply:
this will require a bit of testing once there's a preview build that works